R/BenchmarkResult.R

```diff
@@ -215,6 +218,7 @@ BenchmarkResult = R6Class("BenchmarkResult",
     #'
     #' @return [data.table::data.table()].
     aggregate = function(measures = NULL, ids = TRUE, uhashes = FALSE, params = FALSE, conditions = FALSE) {
+
```

Comment:
I'm not a big fan of enforcing a newline here. Any reason to keep this?

Reply:
I can't find a reference anymore in the style guide but at some point the idea was to leave a blank line when the function declaration is long. Assuming you have a function declaration somewhere that has a blank line at the top, would you want to remove it or simply not touch it? If we don't touch it, we also can't touch 
```r
{ 
  # code 
}
``` 
easily (because of the inner workings of styler). A compromise would be to allow up either one or zero blank lines in any of these two curly brace expressions, but not more.

Reply:
> I can't find a reference anymore in the style guide

The style guide might also not be 100% up-to-date 😅 

I think being deliberate here is probably best - i.e. allowing both.

> A compromise would be to allow up either one or zero blank lines in any of these two curly brace expressions, but not more.

Sounds like a good compromise to me (if that is easier for {styler} also). @mllg?

I think we never want to have two blank lines anywhere and any of zero or one is fine?